first-test-go/main.go

```diff
@@ -0,0 +1,5 @@
+package main
+
+func main() {
```

Comment:
Is this file even needed?

Reply:
not really, my IDE made it up :( removed